### PR TITLE
Defending Ring

### PR DESCRIPTION
**Pull request by RunAwayDSP**

### CODE REVIEW

File: scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua [hunk 1]
```diff
@@ -23,7 +23,14 @@ end;
 function onMobDeath(mob, killer)
 
     killer:addTitle(BEHEMOTH_DETHRONER);
-
+	
+	if(math.random((1),(100)) <= 5) then -- Hardcoded "this or this item" drop rate until implemented.
+		SetDropRate(1936,13566,1000); -- Defending Ring
+		SetDropRate(1936,13415,0);
+    else
+		SetDropRate(1936,13566,0);
```
Comment (TeoTwawki) [inline scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua]: Just so you know, mixing tabs and spaces makes display look unaligned even when its technically correct. That's why github has it looking different than in your editor.
isMagic

(you don't really have to change anything, just letting you know about helpfull features here)

**Comment by RunAwayDSP** [comment]
thanks you for your knowledge master

**Comment by TeoTwawki** [comment]
You are welcome. 
-***pats head***-

**Comment by teschnei** [comment]
i hate it, but i'll allow it for now

**Pull request merged**